connect/resource_kafka_connector.go

```diff
@@ -52,19 +51,11 @@ func setNameFromID(d *schema.ResourceData, meta interface{}) ([]*schema.Resource
 }
 
 func connectorCreate(d *schema.ResourceData, meta interface{}) error {
-	c := meta.(kc.Client)
+	c := meta.(kc.HighLevelClient)
 	name := nameFromRD(d)
 
 	config, sensitiveCache := configFromRD(d)
-	if !kc.TryUntil(
```

Comment:
Why did this get removed?

Reply:
The go-kafka-connect client has changed and no longer exports `TryUntil`. Also the other operations `connectorUpdate` etc. didn't have this extra connection attempt so I removed it rather than implemented an alternative.

The go-kafka-connect client has similar functionality in effect since it is called with `sync=true` in `c.CreateConnector(req, *true*)`, see: https://github.com/ricardo-ch/go-kafka-connect/blob/2.1.0/lib/connectors/highlevel_client.go#L91